### ISSUE TITLE
Title bar does not load properly on Windows Vista

### ISSUE THREAD

**Issue by CaptainTER06** [IDA-RE-things/Chrome-xp-api-adapter#12]
Hello, I'm using this DLL with Supermium 122 R5 on Windows Vista (32-bit), and the title bar is not loading properly (no controls on the right - Minimize, Maximize, Close - and a very buggy looking title bar). 

Could you please fix it?

**Comment by IDA-RE-things** [comment]
this is under developmemnt while. Its not for use. I will remove info avout Vista/Win7 support, while its in development. Currently officially supported only NT5 systems.

**Comment by IDA-RE-things** [comment]
I will close this issue while, because Win7/Vista support is under development while, having continuous changes etc.
So users should test it later, not now :)

**Comment by IDA-RE-things** [comment]
I will close this issue while, because Win7/Vista support is under development while, having continuous changes etc.
So users should test it later, not now :)

**Comment by IDA-RE-things** [comment]
New version available. You can test it now.

**Comment by CaptainTER06** [comment]
Thanks, but where do I download it? Publish it as release please, otherwise it's not easy to find in all the pages available on Github xD

**Comment by IDA-RE-things** [comment]
In Releases section as usual. It published only there always :)

**Comment by IDA-RE-things** [comment]
https://github.com/IDA-RE-things/Chrome-xp-api-adapter/releases

**Comment by CaptainTER06** [comment]
I've tested it, the controls still do not show. I've renamed the "Vista" thing to "progwrp.dll" and placed the other file around it. It it still buggy. Does it require "--no-sandbox"?

**Comment by CaptainTER06** [comment]
With a theme from the Chrome Web Store, the controls show up, but not the aero ones.

**Comment by IDA-RE-things** [comment]
> the controls still do not show.

Thats strange... Becuse now all UI elements should be handled by system DLLs.
Can you create screenshot ?

No, --no-sandbox not requered.

**Comment by IDA-RE-things** [comment]
So with original progwrp all elements shows properly ? Only in this case its a bug by my dll, and I will investigate it.
But I need a screenshot whats happened.

**Comment by IDA-RE-things** [comment]
Becasue I have seen several problems in issues, on supermium official issue tracker, touching Header/Buttons. May be its from there. You should compare with original lib.
May be its a problem inside of browser code itself.

**Comment by CaptainTER06** [comment]
> So with original progwrp all elements shows properly ? Only in this case its a bug by my dll, and I will investigate it. But I need a screenshot whats happened.

Yeah, the with the original DLL, everything works fine. I'll post a screenshot soon for you to understand my issue.

**Comment by IDA-RE-things** [comment]
Should be fixed in my today update 5056.

![2704_wrDoc_WRClipping39636](https://github.com/IDA-RE-things/Chrome-xp-api-adapter/assets/9424365/3e8eca12-d902-43fe-8750-cf35e7a74a95)

**Comment by CaptainTER06** [comment]
Gonna try it, thanks. Except making it run faster on HDDs, what other things does this library ?

**Comment by IDA-RE-things** [comment]
For Vista I think nothing. Its just my implementation of same API functions.
Its not the whole browser code to expect something impossible from this replacement DLL :)
Other optimizations -- should be there.

**Comment by CaptainTER06** [comment]
I've tested it with my Windows Vista laptop, it works very well! Would it be possible to add even more optimizations to the the browser/DLL, or what is in the DLL is currently the maximum that can be done?
